src/audio/pcm_converter/pcm_converter_generic.c

```diff
@@ -1,14 +1,17 @@
 // SPDX-License-Identifier: BSD-3-Clause
-//
-// Copyright(c) 2019 Intel Corporation. All rights reserved.
-//
-// Author: Tomasz Lauda <tomasz.lauda@linux.intel.com>
+/*
```

Comment:
IIRC, the SPDX spec uses //

Reply:
We seem to have adopted the following following rules in this project:

Start .c files like this:
```
// SPDX-License-Identifier: ...
/*
 * Copyright(c) ...
 *
 * Author: ...
 */

Reply:
Fair enough, I must be thinking of something else.

Reply:
It's a bit complicated but I tried to explain it in commit 1db950e0780cf7